ui/page/playlists/internal/collectionsListMine/view.jsx

```diff
@@ -56,20 +64,26 @@ export default function CollectionsListMine(props: Props) {
 
   const unpublishedCollectionsList = (Object.keys(unpublishedCollections || {}): any);
   const publishedList = (Object.keys(publishedCollections || {}): any);
+  const editedList = (Object.keys(editedCollections || {}): any);
+  const savedList = (Object.keys(savedCollections || {}): any);
   const collectionsUnresolved = unpublishedCollectionsList.length === 0 && publishedList.length === 0 && hasCollections;
   const playlistShowCount = isMobile ? COLS.PLAYLIST_SHOW_COUNT.MOBILE : COLS.PLAYLIST_SHOW_COUNT.DEFAULT;
 
   const collectionsToShow =
     React.useMemo(() => {
       switch (filterType) {
         case COLS.LIST_TYPE.ALL:
-          return unpublishedCollectionsList.concat(publishedList);
+          return [...unpublishedCollectionsList, ...publishedList, ...savedList];
         case COLS.LIST_TYPE.PRIVATE:
           return unpublishedCollectionsList;
         case COLS.LIST_TYPE.PUBLIC:
           return publishedList;
+        case COLS.LIST_TYPE.EDITED:
+          return editedList;
+        case COLS.LIST_TYPE.SAVED:
+          return savedList;
       }
-    }, [filterType, publishedList, unpublishedCollectionsList]) || [];
+    }, [editedList, filterType, publishedList, savedList, unpublishedCollectionsList]) || [];
```

Comment:
:arrow_down: Just a discussion

- Considering there could be users with large number of lists, I wonder if concat is faster.
  -  https://stackoverflow.com/a/57191593/977819. 
  - https://www.measurethat.net/Benchmarks/Show/579/1/arrayprototypeconcat-vs-spread-operator
- The `switch :: default` could return the `[ ]` rather than using an explicit fallback outside of `memo` (minor).

Reply:
Good points 👍🏻